webapp/src/Constants.ts

```diff
@@ -50,5 +50,4 @@ export const Constants = {
         MANIFEST_PATH: '/.well-known/ai-plugin.json',
     },
     KEYSTROKE_DEBOUNCE_TIME_MS: 250,
-    STEPWISE_RESULT_NOT_FOUND_REGEX: /(Result not found, review _stepsTaken to see what happened\.)\s+(\[{.*}])/g,
```

Comment:
Doesn't look related but I'm assuming that's fine?

Reply:
Yeah I moved it to the strings file in a previous PR, this was the result of merging main